Python/Product/Profiling/Profiling/ProfiledProcess.cs

```diff
@@ -112,6 +113,10 @@ sealed class ProfiledProcess : IDisposable {
 
             string perfMonPath = Path.Combine(perfToolsPath, "VSPerfMon.exe");
 
+            if (!File.Exists(perfMonPath)) {
+                throw new InvalidOperationException("Cannot locate performance tools.");
```

Comment:
There are already InvalidOperationExceptions raised from this function, and clearly something is catching the Win32Exception that currently gets thrown below. I don't see any explicit catches in the call chain, so guessing it's a protected function via the commanding system.